extensions-compose-jetbrains/src/commonMain/kotlin/com/arkivanov/decompose/extensions/compose/jetbrains/stack/animation/Slide.kt

```diff
@@ -2,22 +2,54 @@ package com.arkivanov.decompose.extensions.compose.jetbrains.stack.animation
 
 import androidx.compose.animation.core.FiniteAnimationSpec
 import androidx.compose.animation.core.tween
+import androidx.compose.foundation.gestures.Orientation
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.layout.layout
 
 /**
  * A simple sliding animation. Children enter from one side and exit to another side.
  */
-fun slide(animationSpec: FiniteAnimationSpec<Float> = tween()): StackAnimator =
+@Deprecated(
+    message = "Please use the new slide StackAnimator with orientation specification",
+    replaceWith = ReplaceWith(
```

Comment:
`replaceWith` shouldn't be required, because the method is hidden and can't be called. The new method will be used automatically.